scripts/cvmfs/setup-nightlies.sh

```diff
@@ -36,6 +36,7 @@ function list_releases() {
             compiler="gcc14"
         fi
     elif [ "$os" = "ubuntu22" ]; then
+        echo "Warning: Ubuntu 22.04 is deprecated, please use Ubuntu 24.04"
```

Comment:
```suggestion
        echo "Warning: Ubuntu 22.04 is deprecated, please use Ubuntu 24.04 or Alma9"
```
Just to point out there are also other options?

Reply:
Yes, added Alma 9 in all the new messages